Apps/W1/DynamicsGPHistoricalData/app/src/Tables/HistMigrationStepError.Table.al

```diff
@@ -0,0 +1,69 @@
+table 40911 "Hist. Migration Step Error"
```

Comment:
Maybe this functionality should go to the Cloud migration app. Steps and errors are applicable only during migration. 
Feel free to wontfix if you disagree.